packages/DatePicker/src/helpers.js

```diff
@@ -1,5 +1,4 @@
 import React from "react";
-import momentPropTypes from "react-moment-proptypes";
 
 export function extractChildrenProps(children, target) {
```

Comment:
This is also a nice, generic helper we could use elsewhere (I think). Can we also migrate it to the `helpers` package?

Reply:
@mikrotron Good idea. But I have question. After I extract this function to `helpers` package, should I bump version in `packages/helpers/package.json` and declare this version as minimal required version in `packages/DateInput/package.json`?

Reply:
we already have a generic way for extracting the children on the helpers, I remember @allison-c  modify that one in this component because we didn't have it on our helper's functions. 

https://github.com/acl-services/paprika/blob/master/packages/helpers/src/extractChildren.js

Reply:
@DanilAgafonov Don't worry, `lerna` should take care of all the versioning for us when we publish it!